reflex/state.py

```diff
@@ -484,6 +484,23 @@ def get_current_page(self) -> str:
         """
         return self.router_data.get(constants.RouteVar.PATH, "")
 
+    def get_page_crumbs(self) -> List[Tuple[str, str]]:
```

Comment:
Maybe this method can be called `get_path_parts` ? Crumbs (without breadcrumbs) is a bit confusing